packages/styles/schematics/utils/dom/update.ts

```diff
@@ -0,0 +1,6 @@
+import { Cheerio, CheerioAPI } from 'cheerio';
+
+export default interface IDomUpdate {
+    selector: string;
+    update: ($elements: Cheerio<any>, $: CheerioAPI) => void;
```

Comment:
```suggestion
    update: ($elements: Cheerio<unknown>, $: CheerioAPI) => void;
```

Reply:
This is defined by cheerio and therefore it's not possible to make this change, because it breaks the functionality.